src/containers/blogs/Blogs.js

```diff
@@ -1,11 +1,30 @@
-import React, {useContext} from "react";
+import React, {useState, useEffect, useContext} from "react";
 import "./Blog.scss";
 import BlogCard from "../../components/blogCard/BlogCard";
 import {blogSection} from "../../portfolio";
 import {Fade} from "react-reveal";
 import StyleContext from "../../contexts/StyleContext";
 export default function Blogs() {
   const {isDark} = useContext(StyleContext);
+  const [fetchedBlogs, setBlogs] = useState([]);
+  useEffect(() => {
+    if (blogSection.mediumUserAccount !== "")
+      fetch(
+        "https://api.rss2json.com/v1/api.json?rss_url=https://medium.com/feed/@" +
```

Comment:
Rather than relying on a 3rd party service at runtime, I would like to get the data via [fetch.js](https://github.com/saadpasta/developerFolio/blob/master/fetch.js) and save it in say `/posts.json` and load that file at run time. That's how it's done for the project section(see https://github.com/saadpasta/developerFolio/pull/333).